### DISCUSSION TITLE
Twitch加速似乎不行

### DISCUSSION THREAD

**Discussion by MCTVVTCM** [BeyondDimension/SteamTools#389] [Q&A]
当我打开Twitch.com时，他给了我"CardSavr Resource Not Found"我该怎么样解决

**Comment by elarbor** [comment]
vercel.app 的也不成功
![image](https://github.com/user-attachments/assets/e81f2144-d9a8-474e-ab28-8f2c5f2a8339)
![image](https://github.com/user-attachments/assets/022177de-237a-4071-9174-13d1cd50ac9d)
![image](https://github.com/user-attachments/assets/7e3d097b-4b57-4858-a691-9f429402cfac)
